src/category_theory/abelian/exact.lean

```diff
@@ -191,6 +191,40 @@ lemma epi_iff_cokernel_π_eq_zero : epi f ↔ cokernel.π f = 0 :=
 
 end
 
+section opposite
+
+instance exact.op [exact f g] : exact g.op f.op :=
+begin
+  rw exact_iff,
+  refine ⟨by simp [← op_comp], _⟩,
+  apply_fun quiver.hom.unop using quiver.hom.unop_inj,
+  simp only [unop_comp, cokernel.π_op, eq_to_hom_refl, kernel.ι_op, category.id_comp,
+    category.assoc, kernel_comp_cokernel_assoc, zero_comp, comp_zero, unop_zero],
+end
+
+lemma exact.op_iff : exact g.op f.op ↔ exact f g :=
+⟨λ e, begin
+  rw exact_iff at e ⊢,
+  refine ⟨by convert (congr_arg quiver.hom.unop e.1), _⟩,
+  have e3 := ((cokernel_op_unop g).inv ≫= congr_arg quiver.hom.unop e.2) =≫ (kernel_op_unop f).hom,
+  simp only [← category.assoc, iso.inv_hom_id, unop_comp, cokernel.π_op, kernel.ι_op,
+    category.id_comp, eq_to_hom_refl, category.comp_id, unop_zero, comp_zero, zero_comp] at e3,
+  simpa only [category.assoc, iso.inv_hom_id, category.comp_id] using e3,
```

Comment:
You can golf this proof by first proving that equivalences preserve exactness. See the sketch that I posted.